_replication/scheduling/advanced-cron-scheduling.md

```diff
@@ -0,0 +1,334 @@
+                </td>
+            </tr>
+        {% endfor %}
+        </table>
+    </div>
+</div>
+<br>
+
+### Specifying days in cron expressions {#specifying-days}
+
+Stitch's implementation of cron uses [POSIX](http://pubs.opengroup.org/onlinepubs/7908799/xcu/crontab.html){:target="new"}, which, unlike other cron libraries, uses a unique method to handle the specification of days. This section will focus on the behavior of the `day of month`, `month`, and `day of week` fields.
+
+{% include note.html content="**Field restriction**: In this section, _restricted_ means that a field contains anything other than an asterisk. An asterisk will select all values for a field." %}
+
+In POSIX, the specification of days is made using the `day of month` and `day of week` fields. How days are specified depends on which fields are restricted. In the table below are explanations of how each field combination will behave, example expressions, and the schedule the expression will translate to.
+
+<table class="attribute-list">
+    <tr>
+        <td width="35%; fixed">
+            <strong>
+                Explanation
+            </strong>
+        </td>
+        <td width="30%; fixed">
+            <strong>
+                Expression
+            </strong>
+        </td>
+        <td>
+            <strong>
+                Translation
+            </strong>
+        </td>
+    </tr>
+    {% for explanation in cron.day-specification-examples %}
+        {% for example in explanation.examples %}
+            {% capture total-examples %}
+                {{ forloop.length }}
+            {% endcapture %}
+        {% endfor %}
+
+        <tr>
+            <td rowspan="{{ total-examples | strip }}">
+                {{ explanation.explanation | markdownify }}
+            </td>
+
+            {% for example in explanation.examples %}
+                {% case forloop.first %}
+                    {% when true %}
+                            <td>
+                                {{ example.expression | markdownify | strip }}
+                            </td>
+                            <td>
+                                {{ example.translation | strip }}
+                            </td>
+                        </tr>
+                    {% else %}
+                        <tr>
+                            <td>
+                                {{ example.expression | markdownify | strip }}
+                            </td>
+                            <td>
+                                {{ example.translation | strip }}
+                            </td>
+                        </tr>
+                {% endcase %}
+            {% endfor %}
+    {% endfor %}
+</table>
+
+
+---
+
+## Example replication schedules {#examples}
+
+{% capture resource-callout %}
+**Want to test an expression?** Use [{{ cron.resource-names.crontab }}]({{ cron.resource-urls.crontab }}){:target="new"} to test expressions and see the schedule that will be created.
```

Comment:
I think it's important to note here that crontab.guru implements several extensions that are not strictly posix compliant. So just because crontab.guru shows you the execution schedule of the cron expression doesn't mean that it will work in stitch.

Reply:
Good call - I'll add that in.